patterngen/MsgPattern[] pattern_list
string units	# 'volts' or 'millimeters'

